### DISCUSSION TITLE
None of the react-testing-lib exemples works 😔

### DISCUSSION THREAD

**Discussion by MohamedAmineBoufares** [vitest-dev/vitest#3858] [Q&A]
Hi everyone,

Just wanted to use Vitest with react-testing-library in a React-Vite project. But before that, I wanted to have a look at the examples provided by the team. I tried manly the "react-testing-lib" example, but It keeps poping this error in the Vitest-ui:

![image](https://github.com/vitest-dev/vitest/assets/68915231/a03e831b-799d-4fec-b04c-fbc994807906)

I haven't changed a thing from [the cloned repository](https://github.com/vitest-dev/vitest/tree/main/examples/react-testing-lib).

All I did was:
```bash
pnpm i

pnpm test:ui
```

Used version:
* node: 18.14.0
* npm: 9.8.1
* pnpm: 8.6.11

I really need this to work, so I won't face any problems when I integrate Vitest into the finale project. Jest is not an option, well it's the latest of options. I really like the UI, it helps us a lot as a team to interact and have stats about our tests.

**Comment by sheremet-va** [comment]
Looks like the problem with UI package in general

**Reply by MohamedAmineBoufares** [reply]
I hope it gets a fix soon...
